src/components/UserDetails.vue

```diff
@@ -136,6 +144,29 @@ export default {
         .catch(e => {
           console.log(e);
         });
+    },
+    // the IDP name in keycloak doesn't always match what's known by users
+    // format to match standard naming or capitalization
+    formatIdentityProvider: function(idp) {
```

Comment:
I think this would be nicer as a computed property but I'm not sure that's possible with an array that will be used in a for loop.

Reply:
Something to consider is a "filter". I used on in the AdminEventLog.

https://github.com/bcgov/moh-keycloak-user-management/blob/626bb546741d90d0e67854af9ee6732eef982d3d/src/views/AdminEventLog.vue#L71

Reference: https://vuejs.org/v2/guide/filters.html

Reply:
thanks i'll look into this